plans/002-fix-doc-drift.md

```
@@ -0,0 +1,131 @@
+- `docs/claude/architecture.md` line 7: `Four collections defined in ``src/content/config.ts``:` — that file does not exist; the real file is `src/content.config.ts` (repo root `src/`, note the dot, not a slash).
+- `docs/environment-setup.md` — contains zero mentions of Resend or `RESEND_API_KEY` (verified via `grep -in 'resend' docs/environment-setup.md` → no matches). It documents only Cloudinary and Notion vars.
+- No `.env.example` exists (`ls .env.example` → not found). A live `.env` exists locally but is gitignored — **never read, copy, or quote `.env` contents**.
+- CI (`.github/workflows/actions.yaml:24-28`) additionally passes `CLOUDINARY_API_SECRET` and `CLOUDINARY_API_KEY` to the build.
+
+## Commands you will need
+
+| Purpose | Command | Expected on success |
+|---|---|---|
+| Find remaining wrong versions | `grep -rn "Astro 5" CLAUDE.md README.md docs/` | no matches (after fix) |
+| Find wrong config path | `grep -rn "src/content/config.ts" docs/ CLAUDE.md README.md` | no matches (after fix) |
+| Enumerate env vars used by code | `grep -rhoE "(import\.meta\.env|process\.env)\.[A-Z_]+" src scripts vendor --include='*.ts' --include='*.tsx' --include='*.astro' \| grep -oE "[A-Z_]+$" \| sort -u` | list of var names |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)